Orin_Flash/jetson-orin-nano-seeed-j3010.md

```diff
@@ -0,0 +1,61 @@
+# Instructions for the Seeed reComputer J3010
```

Comment:
Perhaps we could add 4GB here, like we do for the J4012. Or drop 16GB from J4012? The dashboard name is J3010 4GB